src/nvme.rs

```diff
@@ -88,14 +111,16 @@ pub fn list_nvme_controllers() -> BlockResult<Vec<String>> {
 }
 
 /// List the nvme devices on the host
-pub fn list_nvme_devices() -> BlockResult<Vec<String>> {
-    let out = Command::new("nvme-list").args(&["-o", "json"]).output()?;
+pub fn list_nvme_devices() -> BlockResult<Vec<NvmeDevice>> {
```

Comment:
Nice find. Yeah I made this on a much older version of ubuntu. It's possible the commands have changed since then.